l10ninsp/slave.py

```diff
@@ -123,6 +132,21 @@ def finished(self, *args):
       rc = FAILURE
     self.sendStatus({'rc': rc})
 
+def compareDirs(path1, path2):
+    curPath = os.path.dirname(os.path.abspath(__file__))
+    vendorLocalPath = os.path.join(curPath, '..', '..')
+    cljsPath = os.path.join(vendorLocalPath, 'compare-locales.js')
+    # nodejs ./bin/compare-dirs.js --data=json ~/elmo/stage/workdir/a
+    process = subprocess.Popen(
+      ["nodejs",
```

Comment:
This will still only work in Debian/Ubuntu where the binary is called `nodejs`. On CentOS  and Mac OS X it may be called `node` unfortunately.